README.md

```diff
@@ -18,7 +18,7 @@ Copyright (c) 2018 SunSpec Alliance
 
 
 # Requirements
-- Python 2.7, 3.3-3.6
+- Python 2.7, 3.3-3.7
```

Comment:
The same info needs updating in doc/pysunspec.rst

Reply:
Thanks for catching that.